sample/src/main/java/jp/pokepay/pokepay/LowLevelAPITests.java

```diff
@@ -203,7 +203,7 @@ public String CashtrayTest() throws BankRequestError, ProcessingError {
             throw new Error("attempts should be 0");
         }
         System.out.println("決済しようとする => 失敗");
-        CreateTransactionWithCashtray createTransactionWithCashtray = new CreateTransactionWithCashtray(cashtray.id, null,null);
+        CreateTransactionWithCashtray createTransactionWithCashtray = new CreateTransactionWithCashtray(cashtray.id, null,null, null);
```

Comment:
This is just fixing the tests in the sample project.  This won't affect our SDK.